x-pack/plugins/timelines/public/components/t_grid/standalone/index.tsx

```diff
@@ -237,7 +237,6 @@ const TGridStandaloneComponent: React.FC<TGridStandaloneProps> = ({
     loading,
     { events, updatedAt, loadPage, pageInfo, refetch, totalCount = 0, inspect },
   ] = useTimelineEvents({
-    alertConsumers,
```

Comment:
I believe this should still be populated and it should be observability since standalone is for them? @XavierM correct?